### ISSUE TITLE
Can't select an arbitrary deck when sharing an image to Image Occlusion

### ISSUE THREAD

**Issue by snowtimeglass** [ankidroid/Anki-Android#15612]
###### Reproduction Steps

1. On web browser app (e.g. Chrome), use `Share image` ->`Image Occlusion`
    <img src="https://github.com/ankidroid/Anki-Android/assets/10436072/9fc805ed-09d7-40ab-9200-b73f25939cec" width="320px">

###### Expected Result
Can choose a deck you want to put the card (with the image) in

###### Actual Result
In the card adding process, no option to select a deck. 
    <img src="https://github.com/ankidroid/Anki-Android/assets/10436072/097f5c3c-12c1-405a-8780-ef83c938bd8c" width="320px">

(Supposedly,) the cards is put in the last selected deck after 'Add' button is tapped.

It might be desirable to open `Add` screen before or after Image Occlusion editor screen, in future.

###### Debug info
AnkiDroid Version = 2.17beta6 (c17fed63b97a464484f7ee465057ff251f54007f)

Backend Version = 0.1.34-anki23.12.1 (23.12.1 1a1d4d5419c6b57ef3baf99c9d2d9cf85d36ae0a)

Android Version = 11 (SDK 30)

ProductFlavor = full

Manufacturer = OPPO

Model = CPH2013

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 11; CPH2013 Build/RKQ1.200903.002; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/121.0.6167.165 Mobile Safari/537.36

ACRA UUID = df169a4d-29ec-46b6-b114-141ca7e01ba3

Crash Reports Enabled = true

AnkiDroid Version = 2.17beta5 (a06d445bfcea3139accc30faa1b24c0fe214d2b0)

Backend Version = 0.1.34-anki23.12.1 (23.12.1 1a1d4d5419c6b57ef3baf99c9d2d9cf85d36ae0a)

Android Version = 11 (SDK 30)

ProductFlavor = full

Manufacturer = OPPO

Model = CPH2013

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 11; CPH2013 Build/RKQ1.200903.002; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/121.0.6167.165 Mobile Safari/537.36

ACRA UUID = df169a4d-29ec-46b6-b114-141ca7e01ba3

Crash Reports Enabled = true

###### Research
*Enter an [x] character to confirm the points below:*

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by criticalAY** [comment]
It can be an enhancement but I didn't do it on purpose as it would have increased the number of steps which was not the purpose  of this but now that you have pointed it out we can consider it (would need a workaround, quite some work)

**Comment by Devaprasad-reddy** [comment]
Yep, I've been using the native IO lately and I've been changing the mistakenly added group of cards to the deck i actually wanted to add.
A select deck option in the top bar or just beside the add button would solve this issue as it is in the anki doodle app. It does help...
![Screenshot_2024-02-28-20-40-07-28_c8225022784d0ddd7b2e581ccbb70fc6](https://github.com/ankidroid/Anki-Android/assets/36725943/ec129410-8585-4242-86a3-7d6e633749d9)
![Screenshot_2024-02-28-20-40-10-31_c8225022784d0ddd7b2e581ccbb70fc6](https://github.com/ankidroid/Anki-Android/assets/36725943/3eac7073-1f22-44a3-892f-e8a2055891a2)

**Comment by rodrigo-morales-1** [comment]
@criticalAY  I created a post on Anki Forums related to this topic. You can find it here: [Modify GUI to allow changing deck and editing any field of Image Occlusion of an image "Send to..." AnkiDroid - AnkiDroid (Android) / Suggestions - Anki Forums](https://forums.ankiweb.net/t/feature-request-modify-gui-to-allow-changing-deck-and-editing-any-field-of-image-occlusion-of-an-image-send-to-ankidroid/66472)
